.github/workflows/pr-open.yml

```diff
@@ -1,14 +1,13 @@
 name: Pull Request
 
-on:
+on: 
+  push:
+    branches: [main]
+  
   pull_request:
-    branches:
```

Comment:
@Mkankala Can you explain why you changed the trigger for this file? You have modified it so that it triggers when there is a push to main. The file is called "pr-open.yml". What's the intention here?